governance/pyth_staking_sdk/src/utils/transaction.ts

```diff
@@ -20,5 +18,7 @@ export const sendTransaction = async (
     {},
   );
 
-  return sendTransactions(transactions, connection, wallet);
+  for (const transaction of transactions) {
+    await wallet.sendTransaction(transaction.tx, connection);
```

Comment:
remind me why the interface needs a connection?

Reply:
not sure tbh, my guess is that some wallets will use this connection to send the tx instead of their own.